src/reports_api/models/phase_code.py

```diff
@@ -36,23 +36,30 @@ class PhaseCode(db.Model, CodeTable):
     sort_order = Column(Integer())
 
     work_type = relationship('WorkType', foreign_keys=[work_type_id], lazy='select')
-    ea_act = relationship('EAAct',foreign_keys=[ea_act_id], lazy='select')
+    ea_act = relationship('EAAct', foreign_keys=[ea_act_id], lazy='select')
+
+    milestones = relationship("Milestone",
+                              primaryjoin="PhaseCode.id==Milestone.phase_id",
+                              back_populates="phase")
 
     def as_dict(self):
         """Return Json representation."""
         return {
             'id': self.id,
             'name': self.name,
             'sortOrder': self.sort_order,
-            'startEvent': self.start_event,
```

Comment:
can you make these fields using underscore

Reply:
Updated